exporter/otlphttpexporter/config.go

```diff
@@ -14,9 +14,9 @@ import (
 
 // Config defines configuration for OTLP/HTTP exporter.
 type Config struct {
-	confighttp.HTTPClientConfig `mapstructure:",squash"`     // squash ensures fields are correctly decoded in embedded struct.
-	QueueConfig                 exporterhelper.QueueSettings `mapstructure:"sending_queue"`
-	RetryConfig                 configretry.BackOffConfig    `mapstructure:"retry_on_failure"`
+	confighttp.ClientConfig `mapstructure:",squash"`     // squash ensures fields are correctly decoded in embedded struct.
```

Comment:
I think un-embedding is actually unnecessary. The `ClientConfig` field is descriptive enough in the context of `otlphttpexporter` module